test/language/class/class-to-string.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.5
+description: >
+    class to string
+---*/
+class C {}
+assert.sameValue(C.toString(), 'class C {}');
+
+class D { constructor() { 42; } }
+assert.sameValue(D.toString(), 'class D { constructor() { 42; } }');
```

Comment:
I think these tests are overly specific for Test262. From [19.2.3.5 Function.prototype.toString ( )](http://people.mozilla.org/~jorendorff/es6-draft.html#sec-function.prototype.tostring):

> The use and placement of white space, line terminators, and semicolons within the representation String is implementation-dependent.